src/document-fetcher.js

```diff
@@ -81,7 +82,12 @@ function xhrRequest(input, init) {
       // whole document loading to complete. This is fine for the use cases
       // we have now, but may need to be reimplemented later.
       if (xhr.readyState == /* COMPLETE */ 4) {
-        const response = new FetchResponse(xhr);
+        const options = {
+          status: xhr.status,
+          statusText: xhr.statusText,
+          headers: parseHeaders(xhr.getAllResponseHeaders()),
```

Comment:
@jridgewell, we have a pending discussion around this piece of code in one of the earlier PRs, where you wanted to skip this part.

If we want to skip this we'll need to change code in `assetSuccess` and `verifyAmpCorsHeaders` code as they both rely on Response type